extensions/sql-migration/src/main.ts

```diff
@@ -20,10 +20,18 @@ class SQLMigration {
 	async registerCommands(): Promise<void> {
 		const commandDisposables: vscode.Disposable[] = [ // Array of disposables returned by registerCommand
 			vscode.commands.registerCommand('sqlmigration.start', async () => {
-				const connection = await azdata.connection.openConnectionDialog();
-
+				let activeConnection = await azdata.connection.getCurrentConnection();
```

Comment:
Not adding null checks on connectionID so that we can enter the wizard without a valid connection.